public/app/features/dashboard/components/Inspector/PanelInspector.tsx

```diff
@@ -73,15 +83,17 @@ export class PanelInspector extends PureComponent<Props, State> {
     }
 
     // TODO? should we get the result with an observable once?
-    const lastResult = (panel.getQueryRunner() as any).lastResult;
+    const lastResult: PanelData = (panel.getQueryRunner() as any).lastResult;
```

Comment:
@torkelo what's the suggested way here? Accessing private prop feels wrong

Reply:
yea, maybe add a function getLastResult()

Reply:
Yup, this feels better, but I was thinking about observables way. However, maybe that's too simple to use observables.